### ISSUE TITLE
https://cheminfo.github.io/cheminfo-types/ not working

### ISSUE THREAD

**Issue by djeanner** [cheminfo/cheminfo-types#75]
None of the links of the landing page of https://cheminfo.github.io/cheminfo-types/
reach the expected pages.

**Comment by kjappelbaum** [comment]
yes, @lpatiny turned off the build of the json. I don't know if he has plans to replace this with TS docs.
